### ISSUE TITLE
Canot install on postgres 13.10

### ISSUE THREAD

**Issue by Yankzy** [pgvector/pgvector#65]
I'm trying to install postgres vector extension on my mac but am getting

ERROR:  extension "vector" has no installation script nor update path for version "0.4.0".

Here's what I did:

1. Follow the installation guide as shown on [github](https://github.com/pgvector/pgvector):
But when I ran CREATE EXTENSION vector; got an error:
`ERROR:  could not open extension control file "/Applications/Postgres.app/Contents/Versions/13/share/postgresql/extension/vector.control": No such file or directory`

2. I copied the content of pgvector into posgresql/extension using:

sudo cp -r ~/Downloads/pgvector/* /Applications/Postgres.app/Contents/Versions/13/share/postgresql/extension/

Tried running `CREATE EXTENSION vector; ` now the error is:

`ERROR:  extension "vector" has no installation script nor update path for version "0.4.0".`

I am using PostgreSQL 13.10

**Comment by ankane** [comment]
Hey @Yankzy, can you paste the output of:

1. the `make` and `make install` commands (run `make clean` first if needed)
2. the `pg_config` command

Also, you'll want to remove any files manually copied.

**Comment by uabbas** [comment]
It also fails on postgres 15.2.
I installed the extension using conda as described README
Tried running `CREATE EXTENSION vector` which gives the error 
`
ERROR:  extension "vector" is not available
DETAIL:  Could not open extension control file "/usr/share/postgresql/15/extension/vector.control": No such file or directory.
HINT:  The extension must first be installed on the system where PostgreSQL is running.
`
